scripts/processing/06-make_epochs.py

```diff
@@ -123,6 +134,6 @@ def run_epochs(subject_id, tsss=False):
 ###############################################################################
 # Let us make the script parallel across subjects
 
-parallel, run_func, _ = parallel_func(run_epochs, n_jobs=1)
+parallel, run_func, _ = parallel_func(run_epochs, n_jobs=N_JOBS)
```

Comment:
wait, this does not consume humongous memory?

Reply:
what's the memory footprint with something like `n_jobs = 4`?

Reply:
about 1.5 GB per job

Reply:
... oh wait, at the writing stage it does load all epochs at once. (Too bad we can't change this, maybe we should at the MNE level if possible.) That brings it up to be quite large. I'll revert to 1.